javabuilder-authorizer/metrics_reporter.rb

```diff
@@ -11,8 +11,13 @@ def initialize(context)
 
   def log_token_error(status, error_message)
     puts error_message
+    metric_name = TOKEN_STATUS_METRIC_NAMES[status]
+    log(metric_name)
+  end
+
+  def log(metric_name)
```

Comment:
Do we want to add a `puts` here as well (like we have in `log_token_error`) or is logging the error sufficient? Just thinking if we'd want to look this up in our logs for any reason

Reply:
mmm I guess I figured we're logging some detail to DynamoDB at the point of blocking a user/classroom, so that serves as our log in this case. I'm not sure what our best practice is here though.

Reply:
Yeah, totally not required, i just mentioned it since we do have a `puts` statement in the `log_token_error` function for the error message. I'd say fine to ship as is and we can always add in more logging if it becomes necessary?